packages/wcwidth/wcwidth.1.0.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Get terminal display width of a string"
+description: "Get terminal display width of a string"
+maintainer: ["Jonathan Yong"]
+authors: ["Jonathan Yong"]
+license: "MIT"
+homepage: "https://github.com/yongrenjie/wcwidth-ocaml"
+bug-reports: "https://github.com/yongrenjie/wcwidth-ocaml/issues"
+depends: [
+  "dune" {>= "3.3"}
```

Comment:
```suggestion
  "dune" {>= "3.3"}
  "ocaml" {>= "4.14"}
```

The CI fails because of some unfound Stdlib primitives < 4.14.

Alternatively you can depend on a stdlib-compatibility library, but I think >=4.14 is a reasonable constraint.